packages/project-utils/bundling/function/telemetry.js

```diff
@@ -1,54 +1,31 @@
 const fs = require("fs");
-const { https } = require("follow-redirects");
 const telemetry = require("./telemetry");
 const path = require("path");
+const fetch = require("node-fetch");
+const { getProject } = require("@webiny/cli/utils");
 
-const TELEMETRY_BUCKET_URL = process.env.WCP_API_CLIENTS_URL || "d16ix00y8ek390.cloudfront.net";
-
-function requestTelemetryCode() {
-    const options = {
-        method: "GET",
-        hostname: TELEMETRY_BUCKET_URL,
-        path: "/clients/latest",
-        maxRedirects: 20
-    };
-
-    return new Promise((resolve, reject) => {
-        https
-            .request(options, function (res) {
-                const chunks = [];
-
-                res.on("data", chunk => chunks.push(chunk));
-
-                res.on("error", error => reject(error));
-
-                res.on("end", () => {
-                    const body = Buffer.concat(chunks);
-                    resolve(body.toString());
-                });
-            })
-            .end();
-    });
-}
+const WCP_API_CLIENTS_URL =
```

Comment:
craete this from existing env variables WCP_API_URL.
look in: packages/cli/commands/index.js:11